packages/contracts/src/dollar/DirectGovernanceFarmer.sol

```diff
@@ -428,9 +368,9 @@ contract DirectGovernanceFarmer is ReentrancyGuard {
     }
 
     function isMetaPoolCoin(address token) public view returns (bool) {
-        return (token == token2 ||
-            token == token1 ||
-            token == token0 ||
-            token == ubiquityDollar);
+        return (
+            token == token2 || token == token1 || token == token0
+                || token == ubiquityDollar
+        );
     }
 }
```

Comment:
Looks like formatting got messed up on this file. Which one is the correct one?

Reply:
I changed it so each OR statement is on it's own line which meets the style guide.